### PR TITLE
Recommended helm version

### PR DESCRIPTION
**Pull request by workman-Lu**
Recommended helm version

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.2/install.mdx [hunk 1]
```diff
@@ -162,7 +162,11 @@ helm repo update
 2. 安装 KubeVela Core
 
 ```shell script
+#helm >= v3.2.0
 helm install --create-namespace -n vela-system kubevela kubevela/vela-core --version 1.2.2 --wait
+#helm < v3.2.0
+kubectl create vela-system
```
Comment (barnettZQG) [inline i18n/zh/docusaurus-plugin-content-docs/version-v1.2/install.mdx]: kubectl create namespace vela-system

**Comment by barnettZQG** [comment]
@workman-Lu Thanks!  Please change the English doc: https://github.com/oam-dev/kubevela.io/blob/main/docs/install.mdx

**Comment by msftgits** [comment]
[CLA assistant check](https://cla.opensource.microsoft.com/oam-dev/kubevela.io?pullRequest=495) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/oam-dev/kubevela.io?pullRequest=495) before we can accept your contribution.<br/><br/>:x: workman-Lu [sign now](https://cla.opensource.microsoft.com/oam-dev/kubevela.io?pullRequest=495)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/oam-dev/kubevela.io?pullRequest=495) it.</sub>

**Comment by msftgits** [comment]
[CLA assistant check](https://cla.opensource.microsoft.com/oam-dev/kubevela.io?pullRequest=495) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/oam-dev/kubevela.io?pullRequest=495) before we can accept your contribution.<br/><br/>:x: workman-Lu [sign now](https://cla.opensource.microsoft.com/oam-dev/kubevela.io?pullRequest=495)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/oam-dev/kubevela.io?pullRequest=495) it.</sub>

File: i18n/zh/docusaurus-plugin-content-docs/version-v1.2/install.mdx (hunk 1 above)
Comment (workman-Lu) [reply i18n/zh/docusaurus-plugin-content-docs/version-v1.2/install.mdx]: > kubectl create namespace vela-system

Already added

**Review by wonderflow — CHANGES_REQUESTED** [review]
I'm sorry I don't think we should add so many commands just for notice user the helm version.

We should list the helm version that should be `>= v3.2.0` in the requirements

**Comment by workman-Lu** [comment]
> I'm sorry I don't think we should add so many commands just for notice user the helm version.
> 
> We should list the helm version that should be `>= v3.2.0` in the requirements

excellent idea

**Review by wonderflow — APPROVED** [review]
Cool

**Pull request merged**